liquibase-standard/src/main/java/liquibase/snapshot/SnapshotControl.java

```diff
@@ -123,9 +123,9 @@ public String getSerializableFieldNamespace(String field) {
 
     private void setTypes(Set<Class<? extends DatabaseObject>> types, Database database) {
         this.types = new HashSet<>();
-        for (Class<? extends DatabaseObject> type : types) {
-            addType(type, database);
-        }
+        types.stream()
+                .filter(database::supports)
```

Comment:
this filter is new, and basically filters out objects the db does not support

Reply:
Note: I had to update `liquibase.diff.output.changelog.DiffToChangeLogTest#getOrderedOutputTypes_hasDependencies` as a consequence